test/kickstart-templates/kickstart.ks.template

```diff
@@ -47,6 +48,37 @@ if [ -n "REPLACE_PUBLIC_IP" ]; then
     echo "    - REPLACE_PUBLIC_IP" >>/etc/microshift/config.yaml
 fi
 
+# Set the /etc/microshift/lvmd.yaml file to expose the LVM configuration
```

Comment:
Would it make sense to do this part inside the test? I could anticipate other tests that would want other contents for lvmd.yaml, for example, or different storage classes.